qemu_device_creator.py

```diff
@@ -61,6 +61,13 @@ def main():
         "generated source."
     )
 
+    parser.add_argument(
+        "--gen-intermediate-chunk-graphs",
+        action = "store_true",
+        help = "Generate Graphviz files with intermediate graph of chunks "
+        "during header inclusion optimisation for each generated source."
```

Comment:
Maybe the `optimization` is better?
Because the function is called `optimize_inclusions`.
(if change, then correct the name of the commit)